### PR TITLE
new recipe conda-forge for package xsarsea

### PR DESCRIPTION
**Pull request by agrouaze**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by agrouaze** [comment]
Hello @vincelhx  ,
Are you OK to be mentionned in the conda-forge recipe for xsarsea ?

**Comment by vincelhx** [comment]
yes if needed @agrouaze

**Comment by agrouaze** [comment]
Dear @conda-forge/staged-recipes  team, this recipe is ready for review.
Thanks for your help :-)

File: recipes/xsarsea/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xsarsea-{{ version }}.tar.gz
+  sha256: 454e07b312d79f86d2afba05c7cd95cf26e7a406dd852d2c177be3d30db6607f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (ocefpaf) [inline recipes/xsarsea/meta.yaml:14]: This one has scripts and cannot be noarch but if upstream abandon the deprecated script, we can change this later in the feedstock.

Comment (agrouaze) [reply recipes/xsarsea/meta.yaml:14]: Ok to change this. Thanks for the merge.

**Pull request merged**